### PR TITLE
One App Prepare Release

### PR DESCRIPTION
**Pull request by oneamexbot**
This is an auto-generated pull request for One App release changes. Please review this PR and merge it for a release to be initiated. Please close this PR to stop a release from being initiated.

### CODE REVIEW

**Review by Francois-Esquire — APPROVED** [review]

**Review by BenMcLees — DISMISSED** [review]

**Review by JAdshead — DISMISSED** [review]

**Review by JamesSingleton — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,21 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+# [5.11.0](https://github.com/americanexpress/one-app/compare/v5.10.2...v5.11.0) (2021-01-13)
+
+
+### Bug Fixes
+
+* **docs:** rename pwa recipe ([0f2f054](https://github.com/americanexpress/one-app/commit/0f2f0546944e0c5f2a0481daa83f897ec13d294d))
```
Comment (Francois-Esquire) [inline CHANGELOG.md:10]:
```suggestion
```

**Review by Francois-Esquire — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,20 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+# [5.11.0](https://github.com/americanexpress/one-app/compare/v5.10.2...v5.11.0) (2021-01-13)
+
+
+### Bug Fixes
+
+* **tagging/workflow:** use ::set-output over ::set-env ([#390](https://github.com/americanexpress/one-app/issues/390)) ([b6874f7](https://github.com/americanexpress/one-app/commit/b6874f7d1590cfc330d365398687586e81a98a24))
```
Comment (JamesSingleton) [inline CHANGELOG.md:10]: This should not be there either

Comment (JamesSingleton) [inline CHANGELOG.md:10]:
```suggestion
```

**Review by JamesSingleton — APPROVED** [review]

**Review by BenMcLees — APPROVED** [review]

**Pull request merged**